docs/quick-start/single-binary-three.md

```diff
@@ -22,7 +22,7 @@ Want to build from source locally? Add `-build` as an argument to the bootstrap
 
 ## Running TigerBeetle
 
-Now create the TigerBeetle data file for each replica
+Now create the TigerBeetle [data file](../internals/data_file.md) for each replica:
```

Comment:
I am not sure whether we should be linking internal docs from public docs --- they aren't really written for normal users to understand. 

I think it's fine to do that, _but_ we need to make sure that any internal doc that we link in this way starts with the disclaimer that "you don't need to read this to use TB". 

I guess the first-best solution would be to create some kind of glossary entry for the "data file" in the public docs, and, from that entry, to link to internals. 

No need to do anything for this PR though, I think linking is better that not linking at all!

Reply:
Instead of linking like this, we could have an extra sentence that says "(If you are curious, you can learn more about the contents of the data file in the internals doc)"